packages/calling-stateful-client/src/StreamUtils.ts

```diff
@@ -217,13 +217,15 @@ async function createViewLocalVideo(
     return;
   }
 
+  // "Stopping" only happens if the stream was in "rendering" but `disposeView` was called.
+  // Now `createView` has been re-called when can flip the state back to "rendering".
```

Comment:
nit:
// Now that `createView` has been re-called, we can flip the state back to "rendering".